SPIFlashLogger.class.nut

```diff
@@ -286,15 +286,28 @@ class SPIFlashLogger {
     }
 
 
-    function peek() {
```

Comment:
This sort of API modification requires a major version bump according to semver, if we're following it.